libraries/src/CMS/Router/ApiRouter.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\CMS\ApiRouter;
```

Comment:
`namespace Joomla\CMS\Router` (this is mapping your file structure `src/CMS/Router`